Teamleader.php

```diff
@@ -414,6 +414,28 @@ public function crmUpdateContact(
     }
 
     /**
+     * Delete a contact
+     *
+     * @todo    N/A
+     *
+     * @param         $contact can be either an object of type "Contact" or a contact ID
```

Comment:
I'd still use int in the documentation, since strings that can't be converted to integers can't be used.